Assets/Scripts/Utilities/LineOfSightUtilities.cs

```diff
@@ -1,21 +1,23 @@
 using System.Collections.Generic;
 
+using Maes.Map.MapPatrollingGen;
+
 using UnityEngine;
 
 namespace Maes.Utilities
 {
-    public class LineOfSightUtilities
+    public static class LineOfSightUtilities
     {
         // Precompute visibility using an efficient line-drawing algorithm
-        public static HashSet<Vector2Int> ComputeVisibilityOfPoint(Vector2Int start, bool[,] map)
+        public static HashSet<Vector2Int> ComputeVisibilityOfPoint(Vector2Int start, BitMap2D map)
         {
             var visibilitySet = new HashSet<Vector2Int>();
-            for (var x = 0; x < map.GetLength(0); x++)
```

Comment:
Use the variable names x and y instead of width and height

Reply:
Done